SonarQube.Bootstrapper/BuildAgentUpdater.cs

```diff
@@ -62,6 +72,45 @@ public bool TryUpdate(string hostUrl, string targetDir, ILogger logger)
 
         }
 
+        /// <summary>
+        /// Verifies if the logical version of the boostrapper is compatibile with the logical version of the pre/post processors
```

Comment:
Multiple typos.
"Verifies that the pre/post-processors are compatible with this version of the bootstrapper"